Makefile

```diff
@@ -46,13 +47,16 @@ $(LOCAL_BRIDGE_PLUGIN_BINARY): $(SOURCES)
 generate: $(SOURCES)
 	go generate -x ./pkg/... ./plugins/...
 
-.PHONY: unit-test integration-test
+.PHONY: unit-test integration-test e2e-test
 unit-test: $(SOURCES)
 	go test -v -cover -race -timeout 10s ./pkg/... ./plugins/...
 
 integration-test: $(SOURCE)
 	go test -v -tags integration -race -timeout 10s ./pkg/... ./plugins/...
 
+e2e-test:  $(SOURCE) plugins
+	sudo -E CNI_PATH=${ROOT}/bin/plugins ${GO_EXECUTABLE} test -v -tags e2e -race -timeout 60s ./plugins/...
```

Comment:
`PATH` for `sudo` is not necessarily the same as `PATH` for the user running this script (in my case, `go` is not in my global `PATH` but instead lives in `~/.go/bin`).